services/airlock/justfile

```diff
@@ -17,6 +17,7 @@ deploy: && restart
 # restart airlock
 restart *args:
     docker-compose --ansi never up --no-build {{ args }} --detach airlock
+    docker-compose --ansi never up --no-build {{ args }} --detach airlock-file-uploader
```

Comment:
Currently, the autodeployment works by calling `just autodeploy` with systemd timer. This in turn calls `just deploy`, which runs:
```
docker-compose --ansi never pull --quiet airlock
```
Will this restart the airlock-file-uploader? If not, we'll need to add it to the `just deploy` command I think?

This is the part that makes me nervous about having two separate docker containers, as they can get out of sync, and with django migrations, that could get bumpy